python/lsst/ip/isr/isrFunctions.py

```diff
@@ -224,6 +225,70 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         spans.setMask(mask, mask.getPlaneBitMask(maskValue))
 
 
+def maskE2VEdgeBleed(exposure, e2vEdgeBleedSatMinArea=20000,
+                     e2vEdgeBleedSatMaxArea=100000,
+                     e2vEdgeBleedYMax=350, e2vEdgeBleedThreshold=500.,
+                     saturatedMaskName='SAT', log=None):
+    """Mask edge bleeds in E2V detectors.
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure to apply masking to.
+    e2vEdgeBleedSatMinArea : `int`, optional
+        Minimum limit of saturated cores footprint area.
+    e2vEdgeBleedSatMaxArea : `int`, optional
+        Maximum limit of saturated cores footprint area.
+    e2vEdgeBleedYMax: `float`, optional
+        Height of edge bleed masking.
+    e2vEdgeBleedThreshold: `float`, optional
+        Sky background threshold for E2V edge bleed detection.
+    saturatedMaskName : `str`, optional
+        Mask name for saturation.
+    log : `logging.Logger`, optional
+        Logger to handle messages.
+    """
+
+    log = log if log else logging.getLogger(__name__)
+
+    maskedImage = exposure.maskedImage
+    saturatedBit = maskedImage.mask.getPlaneBitMask(saturatedMaskName)
+
+    thresh = afwDetection.Threshold(saturatedBit,
+                                    afwDetection.Threshold.BITMASK
+                                    )
```

Comment:
If you're going to use this sort of multi-line formatting, it should be closer to black style:
```
thresh = afwDetection.Threshold(
    saturatedBit,
    afwDetection.Threshold.BITMASK,
)
```